tests/integration/helpers/cluster.py

```diff
@@ -1397,6 +1444,25 @@ def setup_glue_catalog_cmd(
         )
         return self.base_glue_catalog_cmd
 
+    def setup_hms_catalog_cmd(self, instance, env_variables, docker_compose_yml_dir):
+        self.with_hms_catalog = True
+        self.base_cmd.extend(
+            [
+                "--file",
+                p.join(
+                    docker_compose_yml_dir, "docker_compose_iceberg_hms_catalog.yml"
+                ),
+            ]
+        )
+
+        self.base_iceberg_hms_cmd = self.compose_cmd(
+            "--env-file",
+            instance.env_file,
+            "--file",
+            p.join(docker_compose_yml_dir, "docker_compose_iceberg_hms_catalog.yml"),
+        )
+        return self.base_iceberg_hms_cmd
+
```

Comment:
This looks redundant

Reply:
Fixed in https://github.com/ClickHouse/ClickHouse/pull/83581/commits/fd81d6c302afbfe8c9dd60c35ebca1f5bbc6b87b where I chose only the rabbitmq-related changes